modules/nodes-sigstore-configure-cluster-policy.adoc

```diff
@@ -0,0 +1,223 @@
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1alpha1
+kind: ClusterImagePolicy <1>
+metadata:
+  name: p1
+spec:
+  scopes: <2>
+    - example.com
+  policy: <3>
+    rootOfTrust: <4>
+      policyType: FulcioCAWithRekor <5>
+      fulcioCAWithRekor: <6>
+        fulcioCAData: a2V5RGF0YQ==
+        fulcioSubject:
+          oidcIssuer: "https://expected.OIDC.issuer/"
+          signedEmail: "expected-signing-user@example.com"
+        rekorKeyData: cmVrb3JLZXlEYXRh <7>
+    signedIdentity:
+      matchPolicy: RemapIdentity <8>
+      remapIdentity:
+        prefix: example.com <9>
+        signedPrefix: mirror-example.com <10>
+----
+<1> Creates a `ClusterImagePolicy` object.
+<2> Defines a list of repositories or images assigned to this policy. In a cluster image policy, make sure that the policy does not block the deployment of the {product-title} images in the `quay.io/openshift-release-dev/ocp-release` and `quay.io/openshift-release-dev/ocp-v4.0-art-dev` repositories. Images in these repositories are required for cluster operation.
+<3> Specifies the parameters that define how the images are verified.
+<4> Defines a root of trust for the policy.
+<5> Specifies the policy types that define the root of trust, either a public key or a link:https://docs.sigstore.dev/certificate_authority/overview/[Fulcio certificate]. Here, a Fulcio certificate with required Rekor verification.
+<6> For a Fulcio certificate policy, the following parameters are required:
+* `fulcioCAData`: Specifies a base64-encoded Fulcio certificate in the PEM format. The maximum length is 8192 characters.
+* `fulcioSubject`: Specifies the OIDC issuer and the email of the Fulcio authentication configuration.
+<7> Specifies a base64-encoded Rekor public key in the PEM format. This parameter is required when when the `policyType` is `FulcioCAWithRekor`. The maximum length is 8192 characters.
+<8> Optional: Specifies one of the following processes to verify the identity in the signature and the actual image identity. 
+* `MatchRepoDigestOrExact`.
+* `MatchRepository`.
+* `ExactRepository`. The `exactRepository` parameter must be specified.
+* `RemapIdentity`. The `prefix` and `signedPrefix` parameters must be specified.
+<9> For the `remapIdentity` match policy, specifies the prefix that should be matched against the scoped image prefix. If the two match, the scoped image prefix is replaced with the value of `signedPrefix`. The maximum length is 512 characters.
+<10> For the `remapIdentity` match policy, specifies the image prefix to be remapped, if needed. The maximum length is 512 characters.
+--
+
+. Create the cluster image policy object:
++
+[source,terminal]
+----
+$ oc create -f <file-name>.yaml
+----
++
+The Machine Config Operator (MCO) updates the machine config pools (MCP) in your cluster.
+
+.Verification
+
+. After the nodes in your cluster are updated, you can verify that the cluster image policy has been configured:
```

Comment:
Since this is a single step (with multiple sub-steps), IMO, a bullet can be used here.